src/services/juno/custom/helpers/admin-resource.ts

```diff
@@ -10,14 +10,21 @@ import { customApi } from "../custom";
 export const AP_ID = 0;
 export const REVIEWER_ID = 0.5;
 
+const CONTRACTS = {
+  AP: {
+    cw3: contracts.cw3ApTeam,
+    cw4: contracts.cw4GrpApTeam,
+    type: "ap",
+  },
+  REVIEWER: {
+    cw3: contracts.cw3ReviewTeam,
+    cw4: contracts.cw4GrpReviewTeam,
+    type: "review",
+  },
+};
+
 export function getCWs(id: number) {
-  //TODO: atm, only two admin types, refactor this once > 2
-  //charities doesn't have hardcoded cws, so only test for AP_ID && REVIEWER_ID
-  const cw3 = id === AP_ID ? contracts.cw3ApTeam : contracts.cw3ReviewTeam;
-  const cw4 =
-    id === AP_ID ? contracts.cw4GrpApTeam : contracts.cw4GrpReviewTeam;
-  const type: AdminResources["type"] = id === AP_ID ? "ap" : "review";
-  return { cw3, cw4, type };
+  return isAp(id) ? CONTRACTS.AP : CONTRACTS.REVIEWER;
```

Comment:
we could also get rid of functions `getCWs` and `isAp` entirely

and make use of just the `constant`
```ts
type Admins = AdminResources["type"];
const _charity: Admins = "charity";
type CWs = {
  [key: number]:
    | {
        cw3: string;
        cw4: string;
        type: Exclude<Admins, typeof _charity>;
      }
    | undefined;
};

export const apCWs: CWs = {
  [AP_ID]: {
    cw3: contracts.cw3ApTeam,
    cw4: contracts.cw4GrpApTeam,
    type: "ap",
  },
  [REVIEWER_ID]: {
    cw3: contracts.cw3ApTeam,
    cw4: contracts.cw4GrpApTeam,
    type: "ap",
  },
};
```

and use as
```ts
 const numId = idParamToNum(args.endowmentId);
        const AP = apCWs[numId];
        /** special case for ap admin usage */
        if (AP) {
          const { cw3, cw4, type } = AP;
          //...make use of values
         }
```

Reply:
Implemented aabove change